inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -41,6 +40,9 @@ class UsedCSS extends Table {
 		20220131 => 'make_status_column_index',
 		20220513 => 'add_hash_column',
 		20220920 => 'make_status_column_index_instead_queue_name',
+		20220926 => 'add_error_message_column',
+		20221103 => 'add_error_code_column',
+		20221104 => 'make_error_code_column_index',
```

Comment:
I believe we need to make all of those changes in one function, because of the following reasons:
1. This migrations array is growing too fast
2. Those changes are still not shipped to any customer so no need to make separate function for each one.

Just make it one method called `add_error_columns`

What do u think?